src/components/views/rooms/EntityTile.js

```diff
@@ -156,18 +157,20 @@ const EntityTile = createReactClass({
             );
         }
 
-        let power;
+        let powerLabel;
         const powerStatus = this.props.powerStatus;
         if (powerStatus) {
-            const src = {
-                [EntityTile.POWER_STATUS_MODERATOR]: require("../../../../res/img/mod.svg"),
-                [EntityTile.POWER_STATUS_ADMIN]: require("../../../../res/img/admin.svg"),
-            }[powerStatus];
-            const alt = {
-                [EntityTile.POWER_STATUS_MODERATOR]: _t("Moderator"),
+            const powerText = {
+                [EntityTile.POWER_STATUS_MODERATOR]: _t("Mod"),
```

Comment:
I have a feeling this is not going to translate well at all.

Reply:
Well, I suppose we'll see how it goes... I updated the PR to cap the max-width and use ellipsis if this text gets too long.

Reply:
<img width="243" alt="2020-01-31 at 10 11" src="https://user-images.githubusercontent.com/279572/73531848-3c99c800-4413-11ea-9477-ed8899cce679.png">